docs/extend/modular-resources/_index.md

```diff
@@ -64,7 +64,9 @@ However, you can also create and expose new API types using modular resources.
 
 ## Use a modular resource with your robot
 
-Add a modular resource to your robot configuration in five steps:
+To add a modular resource to your robot configuration, you will sometimes use a modular resource that already exists (that someone else created).
+If this is the case, skip to [Save the executable](#make-sure-viam-server-can-access-your-executable)
+If you are creating your own modular resource, follow these 5 steps:
```

Comment:
```suggestion
If you are using an existing modular resource, you can skip to [Save the executable](#make-sure-viam-server-can-access-your-executable).
If you are creating your own modular resource, follow these steps:
```
- avoid future tense where possible
- shorten (for clarity)
- if it's important who created it we can add "that someone else created" back in
- where possible avoid specifying the number of steps. Easy to forget to update if the number changes.